### PR TITLE
introductory post for sparklyr.sedona

### PR DESCRIPTION
**Pull request by yitao-li**
Signed-off-by: Yitao Li <yitao@rstudio.com>

### CODE REVIEW

**Review by skeydan — APPROVED** [review]
This is very nice! I especially like how you motivate the extension with survey results / comments!

(As far as I remember there were several feature requests regarding spatial data even, though not all related to Spark.)

File: _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  pts_per_region_rdd,
+  resolution_x = 1000,
+  resolution_y = 600,
+  output_location = tempfile("choropleth-map-"),
+  boundary = c(-126.790180, -64.630926, 24.863836, 50.000),
+  base_color = c(63, 127, 255)
+)
+```
+
+which gives us the following:
```
[the remark this answers is not in the document]
Comment (yitao-li) [reply _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd:142]: Actually I'm not sure 🙂 All I know is the data set came from the incubator-sedona repo and contains some polygons.

File: _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+![choropleth map with overlay](images/choropleth-map-with-overlay.png)
+
+With some low-level spatial operations taken care of using the SRDD API and
+the right spatial partitioning and indexing data structures, we can then
+import the results from SRDDs to Spark dataframes. When working with spatial
+objects within Spark dataframes, we can write high-level, declarative queries
+on these objects using `dplyr` verbs in conjunction with Sedona
+[spatial UDFs](https://sedona.apache.org/api/sql/Function/), e.g.
+^[This demo requires sparklyr 1.7 or above to generate the required Spark SQL type casts for `ST_Point()` automatically.],
+
```
[the remark this answers is not in the document]
Comment (yitao-li) [reply _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd:185]: Done.

File: _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+The author of this blog post would like to thank [Jia Yu](https://github.com/jiayuasu),
+the creator of Apache Sedona, and [Lorenz Walthert](https://github.com/lorenzwalthert) for
+their suggestion to contribute `sparklyr.sedona` to the upstream
+[incubator-sedona](https://github.com/apache/incubator-sedona) repository. Jia has provided
+extensive codereview feedback to ensure `sparklyr.sedona` complies with coding standards
+and best practices of the Apache Sedona project, and has also been very helpful in the
+instrumentation of CI workflows verifying `sparklyr.sedona` works as expected with snapshot
+versions of Sedona libraries from development branches.
+
+The author is also grateful for his colleague [Sigrid Keydana](https://github.com/skeydan)
```
[the remark this answers is not in the document]
Comment (yitao-li) [reply _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd:231]: Done.

File: _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+  pt_rdd,
+  polygon_rdd,
+  join_type = "contain",
+  partitioner = "kdbtree"
+)
+
+```
+
+It is worth mentioning that `sedona_spatial_join()` will perform spatial partitioning
+and indexing on the inputs using the `partitioner` and `index_type` specified if the
```
[the remark this answers is not in the document]
Comment (yitao-li) [reply _posts/2021-07-03-sparklyr-sedona/sparklyr-sedona.Rmd]: Yep. Revised accordingly.

**Pull request merged**